src/FilterService/FilterType/SearchIndex/MultiSelect.php

```diff
@@ -86,10 +86,13 @@ public function addCondition(AbstractFilterDefinitionType $filterDefinition, Pro
             if (!empty($quotedValues)) {
                 if ($filterDefinition->getUseAndCondition()) {
                     foreach ($quotedValues as $value) {
-                        $productList->addCondition($value, $field);
+                        $productList->addCondition($value, $isPrecondition ? self::PREFIX_PRECONDITION : $field);
```

Comment:
shouldn't it be like 
```suggestion
                        $productList->addCondition($value, $isPrecondition ? self::PREFIX_PRECONDITION . $field : $field);
```
?

Reply:
Hey @kingjia90  
Looks like I missed two `SearchIndex` classes. I've now updated those and implemented usage of the new `getConditionField` method.  
I've also rebased the feature branch on top of `2.x`.

Reply:
Thank you for the changes!

Could you please rebase on top of the `2.x` again? 
Without https://github.com/pimcore/ecommerce-framework-bundle/pull/260 the tests are failing

Reply:
Rebased.